libs/platform/unit/platform_unit_test.cpp

```diff
@@ -96,9 +96,21 @@ class _test_helper
     bool state_initiated = false;
 };
 
+struct ebpf_hash_table_destroyer_t
+{
+    void
+    operator()(_In_opt_ _Post_invalid_ ebpf_hash_table_t* table)
+    {
+        if (table != nullptr) {
```

Comment:
remove the `if` test.  ebpf_ apis that do frees should always be ok to pass nullptr to, and removing it decreases the number of distinct code paths and hence improves complexity and code coverage.